deploy/ray-operator/controllers/common/service.go

```diff
@@ -0,0 +1,48 @@
+package common
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	rayiov1alpha1 "ray-operator/api/v1alpha1"
+	"ray-operator/utils"
+	"strings"
+)
+
+type ServiceConfig struct {
+	RayCluster rayiov1alpha1.RayCluster
+	PodName    string
+}
+
+func DefaultServiceConfig(instance rayiov1alpha1.RayCluster, podName string) *ServiceConfig {
```

Comment:
maybe `CreateServiceConfig` or `From` is better as the function name.

Reply:
This knowledge from kubemaker